src/Workspaces/Remote/ServiceHub/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "dependencies": {
-        "StreamJsonRpc": "0.12.32-alpha-g90be50f449"
+        "StreamJsonRpc": "1.0.2-rc"
```

Comment:
Yes, @jaredpar and @tmeschter are working on requiring preview 5 in our test environments.